test/e2e/minscale_readiness_test.go

```diff
@@ -49,6 +49,8 @@ func TestMinScale(t *testing.T) {
 		Route:  name,
 		Image:  "helloworld",
 	}
+	test.CleanupOnInterrupt(func() { test.TearDown(clients, names) })
+	defer test.TearDown(clients, names)
```

Comment:
This looks like some rebase weirdness. The respective calls are already in place.

Reply:
done